docs/.eslintrc.json

```diff
@@ -1,4 +1,15 @@
 {
     "extends": "@comet/eslint-config/react",
-    "ignorePatterns": [".docusaurus", "build"]
+    "ignorePatterns": [".docusaurus", "build"],
+    "rules": {
+        "@calm/react-intl/missing-formatted-message": "off",
+        "@typescript-eslint/no-unused-vars": "off"
```

Comment:
The reason I disabled this is that we wrap our stories in a `Story()` function. 

https://github.com/vivid-planet/comet/blob/a9246a503a4c34ee4a009e8beb2f92f00a7d9cef/docs/src/stories/Checkbox.tsx#L7-L15

This function is never called since we use the raw loader for the import. Meaning, the Story function causes eslint to fail. 

If we disable the rule per line, each story would start like this

```tsx
// eslint-disable-next-line @typescript-eslint/no-unused-vars
function Story() {
    const theme = createCometTheme();

    return (
        <MuiThemeProvider theme={theme}>
            <Form
```

We could avoid this by exporting the Story function (although it's not necessary).

Reply:
I'd prefer the unnecessary export over disabling the rule

Reply:
Okay 👍